.github/workflows/test.yml

```
@@ -4,7 +4,11 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

```suggestion
        # os: [ubuntu-latest, windows-latest, macos-latest]  # now macos-latest doesn't work due to their bug, which g++ is not g++. see https://github.com/online-judge-tools/verification-helper/pull/255
        os: [ubuntu-latest, windows-latest]
```

とりあえず macOS を無効にします。
「パス区切りが `/` 以外だと動かない」は我々に責任のあるバグだけど「g++ が g++ 以外だと動かない」は我々に責任はない。矛盾した OS を出荷してる企業か、そういう変な環境を使ってるユーザが頑張るべき